src/app/pages/BuySovPage/components/Input/index.tsx

```diff
@@ -16,6 +10,7 @@ interface Props {
   type: string;
   placeholder?: string;
   invalid?: boolean;
+  dataAttribute?: string;
```

Comment:
I find dataAttribute to be very generic, though its usage is specific to dataActionId, why not just use dataActionId?

Reply:
Aggreed